### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by mchalupa**
See commit messages

### CODE REVIEW

File: process_witness.py [hunk 1]
```diff
@@ -233,6 +233,10 @@ def processWitness(witness, benchmark, bitwidth):
       a = re.sub(r'\\result', '__SV_COMP_result', a)
       # we may be missing typedefs used in type casts
       a = re.sub(r'\([a-zA-Z0-9_]+\s*\*\)', '(int*)', a)
+      # terminate the expression with semicolon (if needed),
+      # so that the parser can correctly parse the code
+      if a[-1] != ';':
+          a += ';'
```
Comment (tautschnig) [inline process_witness.py]: I believe we first need to trim the string to strip off space or just unconditionally always append the ';'.

Nit pick: this code uses 2-space indent (for historic reasons..).

**Review by tautschnig — CHANGES_REQUESTED** [review]

File: process_witness.py (hunk 1 above)
Comment (mchalupa) [reply process_witness.py]: Yes, you're right. The later (unconditionally appending semicolon) seems nicer to me (actually, that way I had it written first, but then I changed it). So can I go with it? Or you want to strip the string?

Comment (tautschnig) [reply process_witness.py]: Yes, go with the unconditional append - I think that should be fine for all the remaining code.

**Comment by tautschnig** [comment]
This will now also require a rebase, please.

**Comment by mchalupa** [comment]
Sorry, I haven't noticed. Done.

**Review by tautschnig — APPROVED** [review]

**Pull request merged**